mizar/dp/mizar/operators/nets/nets_operator.py

```diff
@@ -140,9 +140,10 @@ def on_endpoint_init(self, body, spec, **kwargs):
 
     def allocate_endpoint(self, ep):
         n = self.store.get_net(ep.net)
-        if ep.ip == "":
+        ip = ep.ip
+        if ep.ip == '':
             ip = n.allocate_ip()
-            ep.set_ip(ip)
+        ep.set_ip(ip)
```

Comment:
I am not sure why this change is necessary. It is the same thing. Right?

Reply:
if ip is given, ep.set_ip(ip) will not be reached. So it needs to be move outside of if statement

Reply:
if ep.ip has a value (which is being copied in line 143), then it is already set. Right?